backend/src/near.js

```diff
@@ -25,12 +25,16 @@ const queryFinalBlock = async () => {
 };
 
 const queryEpochStats = async () => {
+  const networkGenesisConfig = await nearRpc.sendJsonRpc(
+    "EXPERIMENTAL_genesis_config",
+    {}
+  );
```

Comment:
Let's move this query (and the dependent data below) inside the `if (currentEpochStartHeight !== epochStartHeight)` to avoid unnecessary queries unless we actually need to update the seat price (once an epoch)